classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
+
```

Comment:
- [ ] For better code organization, kindly move this getter and setter definition to the top of the class.